Failed to start service due to '- is not ready yet.'

I'm on 
OS: Ubuntu 18.04 Desktop
tag: v1.2.0

When I tried to start service by `quick-start-service.sh`, it hangs on `database-controller is not ready yet. Please wait for a moment!` like below:
```
2020-10-15 00:05:45,579 [INFO] - deployment.paiLibrary.paiService.service_management_start : Begin to execute service database-controller's start script.
serviceaccount/database-controller-account created
clusterrolebinding.rbac.authorization.k8s.io/database-controller-role-binding created
statefulset.apps/database-controller-sts created
database-controller is not ready yet. Please wait for a moment!
database-controller is not ready yet. Please wait for a moment!
database-controller is not ready yet. Please wait for a moment!
database-controller is not ready yet. Please wait for a moment!
database-controller is not ready yet. Please wait for a moment!
database-controller is not ready yet. Please wait for a moment!
database-controller is not ready yet. Please wait for a moment!
database-controller is not ready yet. Please wait for a moment!
database-controller is not ready yet. Please wait for a moment!
database-controller is not ready yet. Please wait for a moment!
database-controller is not ready yet. Please wait for a moment!
database-controller is not ready yet. Please wait for a moment!
database-controller is not ready yet. Please wait for a moment!
database-controller is not ready yet. Please wait for a moment!
database-controller is not ready yet. Please wait for a moment!
database-controller is not ready yet. Please wait for a moment!
database-controller is not ready yet. Please wait for a moment!
database-controller is not ready yet. Please wait for a moment!
database-controller is not ready yet. Please wait for a moment!
database-controller is not ready yet. Please wait for a moment!
database-controller is not ready yet. Please wait for a moment!
database-controller is not ready yet. Please wait for a moment!
database-controller is not ready yet. Please wait for a moment!
database-controller is not ready yet. Please wait for a moment!
database-controller is not ready yet. Please wait for a moment!
database-controller is not ready yet. Please wait for a moment!
database-controller is not ready yet. Please wait for a moment!
database-controller is not ready yet. Please wait for a moment!
database-controller is not ready yet. Please wait for a moment!
database-controller is not ready yet. Please wait for a moment!
database-controller is not ready yet. Please wait for a moment!
database-controller is not ready yet. Please wait for a moment!
database-controller is not ready yet. Please wait for a moment!
database-controller is not ready yet. Please wait for a moment!
database-controller is not ready yet. Please wait for a moment!
database-controller is not ready yet. Please wait for a moment!
database-controller is not ready yet. Please wait for a moment!
database-controller is not ready yet. Please wait for a moment!
```

Then I tried to find the cause of this by checking the logs from pods `kubectl logs database-controller-sts-0` and got this:
```
Name:           database-controller-sts-0
Namespace:      default
Priority:       0
Node:           pai-master/172.27.181.14
Start Time:     Thu, 15 Oct 2020 09:05:45 +0900
Labels:         app=database-controller
                controller-revision-hash=database-controller-sts-68cd7d7f75
                statefulset.kubernetes.io/pod-name=database-controller-sts-0
Annotations:    <none>
Status:         Pending
IP:             172.27.181.14
IPs:            <none>
Controlled By:  StatefulSet/database-controller-sts
Init Containers:
  database-initializer:
    Container ID:  docker://e66838fb869e948d55edbb928dacbec836cfe6a2c4e87581ba1e97ef0755f82c
    Image:         openpai/database-controller:v1.2.0
    Image ID:      docker-pullable://openpai/database-controller@sha256:e502a617b8f8614e418daab73d0ca641b53a3b79929ae65f2fd7e77900b5da82
    Port:          <none>
    Host Port:     <none>
    Command:
      node
      --max-old-space-size=8192
      initializer/index.js
    State:          Waiting
      Reason:       CrashLoopBackOff
    Last State:     Terminated
      Reason:       Error
      Exit Code:    1
      Started:      Thu, 15 Oct 2020 10:14:43 +0900
      Finished:     Thu, 15 Oct 2020 10:14:44 +0900
    Ready:          False
    Restart Count:  18
    Environment:
      LOG_LEVEL:                               info
      K8S_CONNECTION_TIMEOUT_SECOND:           120
      WRITE_MERGER_CONNECTION_TIMEOUT_SECOND:  120
      RETAIN_MODE_ENABLED:                     false
      RBAC_IN_CLUSTER:                         true
      DB_CONNECTION_STR:                       postgresql://root:rootpass@172.27.181.14:5432/openpai
    Mounts:
      /var/run/secrets/kubernetes.io/serviceaccount from database-controller-account-token-lnmz8 (ro)
Containers:
  write-merger:
    Container ID:  
    Image:         openpai/database-controller:v1.2.0
    Image ID:      
    Port:          <none>
    Host Port:     <none>
    Command:
      node
      --max-old-space-size=2048
      write-merger/index.js
    State:          Waiting
      Reason:       PodInitializing
    Ready:          False
    Restart Count:  0
    Liveness:       http-get http://:9748/api/v1/ping delay=60s timeout=1s period=60s #success=1 #failure=3
    Readiness:      http-get http://:9748/api/v1/ping delay=0s timeout=1s period=10s #success=1 #failure=3
    Environment:
      LOG_LEVEL:                               info
      K8S_CONNECTION_TIMEOUT_SECOND:           120
      WRITE_MERGER_CONNECTION_TIMEOUT_SECOND:  120
      RETAIN_MODE_ENABLED:                     false
      RBAC_IN_CLUSTER:                         true
      DB_CONNECTION_STR:                       postgresql://root:rootpass@172.27.181.14:5432/openpai
      MAX_DB_CONNECTION:                       100
      PORT:                                    9748
    Mounts:
      /var/run/secrets/kubernetes.io/serviceaccount from database-controller-account-token-lnmz8 (ro)
  framework-watcher:
    Container ID:  
    Image:         openpai/database-controller:v1.2.0
    Image ID:      
    Port:          <none>
    Host Port:     <none>
    Command:
      node
      --max-old-space-size=8192
      watcher/framework/index.js
    State:          Waiting
      Reason:       PodInitializing
    Ready:          False
    Restart Count:  0
    Environment:
      LOG_LEVEL:                               info
      K8S_CONNECTION_TIMEOUT_SECOND:           120
      WRITE_MERGER_CONNECTION_TIMEOUT_SECOND:  120
      RETAIN_MODE_ENABLED:                     false
      RBAC_IN_CLUSTER:                         true
      WRITE_MERGER_URL:                        http://172.27.181.14:9748
      MAX_RPC_CONCURRENCY:                     150
    Mounts:
      /var/run/secrets/kubernetes.io/serviceaccount from database-controller-account-token-lnmz8 (ro)
  poller:
    Container ID:  
    Image:         openpai/database-controller:v1.2.0
    Image ID:      
    Port:          <none>
    Host Port:     <none>
    Command:
      node
      --max-old-space-size=4096
      poller/index.js
    State:          Waiting
      Reason:       PodInitializing
    Ready:          False
    Restart Count:  0
    Environment:
      LOG_LEVEL:                               info
      K8S_CONNECTION_TIMEOUT_SECOND:           120
      WRITE_MERGER_CONNECTION_TIMEOUT_SECOND:  120
      RETAIN_MODE_ENABLED:                     false
      RBAC_IN_CLUSTER:                         true
      DB_CONNECTION_STR:                       postgresql://root:rootpass@172.27.181.14:5432/openpai
      MAX_DB_CONNECTION:                       10
      INTERVAL_SECOND:                         120
      WRITE_MERGER_URL:                        http://172.27.181.14:9748
      MAX_RPC_CONCURRENCY:                     50
    Mounts:
      /var/run/secrets/kubernetes.io/serviceaccount from database-controller-account-token-lnmz8 (ro)
Conditions:
  Type              Status
  Initialized       False 
  Ready             False 
  ContainersReady   False 
  PodScheduled      True 
Volumes:
  database-controller-account-token-lnmz8:
    Type:        Secret (a volume populated by a Secret)
    SecretName:  database-controller-account-token-lnmz8
    Optional:    false
QoS Class:       BestEffort
Node-Selectors:  <none>
Tolerations:     node.kubernetes.io/not-ready:NoExecute for 300s
                 node.kubernetes.io/unreachable:NoExecute for 300s
Events:
  Type     Reason   Age                    From                 Message
  ----     ------   ----                   ----                 -------
  Warning  BackOff  2m30s (x311 over 72m)  kubelet, pai-master  Back-off restarting failed container
```

And also checked the log of initializer container by `kubectl logs database-controller-sts-0 -c database-initializer`:
```
Executing (default): SELECT "id", "version", "commitVersion", "createdAt", "updatedAt" FROM "version" AS "version" LIMIT 1;
Executing (default): CREATE TABLE IF NOT EXISTS "frameworks" ("insertedAt" TIMESTAMP WITH TIME ZONE, "name" VARCHAR(64) , "namespace" VARCHAR(64), "jobName" VARCHAR(256), "userName" VARCHAR(256), "jobConfig" TEXT, "executionType" VARCHAR(32), "creationTime" TIMESTAMP WITH TIME ZONE, "virtualCluster" VARCHAR(256), "jobPriority" VARCHAR(256), "totalGpuNumber" INTEGER, "totalTaskNumber" INTEGER, "totalTaskRoleNumber" INTEGER, "logPathInfix" VARCHAR(256), "submissionTime" TIMESTAMP WITH TIME ZONE NOT NULL, "dockerSecretDef" TEXT, "configSecretDef" TEXT, "priorityClassDef" TEXT, "retries" INTEGER, "retryDelayTime" INTEGER, "platformRetries" INTEGER, "resourceRetries" INTEGER, "userRetries" INTEGER, "completionTime" TIMESTAMP WITH TIME ZONE, "appExitCode" INTEGER, "subState" VARCHAR(32), "state" VARCHAR(32), "snapshot" TEXT, "requestSynced" BOOLEAN NOT NULL DEFAULT false, "apiServerDeleted" BOOLEAN NOT NULL DEFAULT false, "archived" BOOLEAN NOT NULL DEFAULT false, "updatedAt" TIMESTAMP WITH TIME ZONE NOT NULL, PRIMARY KEY ("name"));
Executing (default): CREATE TABLE IF NOT EXISTS "framework_history" ("insertedAt" TIMESTAMP WITH TIME ZONE, "uid" VARCHAR(36) , "frameworkName" VARCHAR(64) NOT NULL REFERENCES "frameworks" ("name") ON DELETE CASCADE ON UPDATE CASCADE, "attemptIndex" INTEGER, "historyType" VARCHAR(16) NOT NULL DEFAULT 'retry', "snapshot" TEXT, "updatedAt" TIMESTAMP WITH TIME ZONE NOT NULL, PRIMARY KEY ("uid"));
Executing (default): CREATE TABLE IF NOT EXISTS "pods" ("insertedAt" TIMESTAMP WITH TIME ZONE, "uid" VARCHAR(36) , "frameworkName" VARCHAR(64) NOT NULL REFERENCES "frameworks" ("name") ON DELETE CASCADE ON UPDATE CASCADE, "attemptIndex" INTEGER, "taskroleName" VARCHAR(256), "taskroleIndex" INTEGER, "taskAttemptIndex" INTEGER, "snapshot" TEXT, "updatedAt" TIMESTAMP WITH TIME ZONE NOT NULL, PRIMARY KEY ("uid"));
Executing (default): CREATE TABLE IF NOT EXISTS "framework_events" ("insertedAt" TIMESTAMP WITH TIME ZONE, "uid" VARCHAR(36) , "frameworkName" VARCHAR(64) NOT NULL REFERENCES "frameworks" ("name") ON DELETE CASCADE ON UPDATE CASCADE, "type" VARCHAR(32) NOT NULL, "message" TEXT, "event" TEXT, "updatedAt" TIMESTAMP WITH TIME ZONE NOT NULL, PRIMARY KEY ("uid"));
Executing (default): CREATE TABLE IF NOT EXISTS "pod_events" ("insertedAt" TIMESTAMP WITH TIME ZONE, "uid" VARCHAR(36) , "frameworkName" VARCHAR(64) NOT NULL REFERENCES "frameworks" ("name") ON DELETE CASCADE ON UPDATE CASCADE, "podUid" VARCHAR(36) NOT NULL, "type" VARCHAR(32) NOT NULL, "message" TEXT, "event" TEXT, "updatedAt" TIMESTAMP WITH TIME ZONE NOT NULL, PRIMARY KEY ("uid"));
Executing (default): CREATE TABLE IF NOT EXISTS "version" ("id"   SERIAL , "version" VARCHAR(36), "commitVersion" VARCHAR(64), "createdAt" TIMESTAMP WITH TIME ZONE NOT NULL, "updatedAt" TIMESTAMP WITH TIME ZONE NOT NULL, PRIMARY KEY ("id"));
Executing (default): SELECT pk.constraint_type as "Constraint",c.column_name as "Field", c.column_default as "Default",c.is_nullable as "Null", (CASE WHEN c.udt_name = 'hstore' THEN c.udt_name ELSE c.data_type END) || (CASE WHEN c.character_maximum_length IS NOT NULL THEN '(' || c.character_maximum_length || ')' ELSE '' END) as "Type", (SELECT array_agg(e.enumlabel) FROM pg_catalog.pg_type t JOIN pg_catalog.pg_enum e ON t.oid=e.enumtypid WHERE t.typname=c.udt_name) AS "special", (SELECT pgd.description FROM pg_catalog.pg_statio_all_tables AS st INNER JOIN pg_catalog.pg_description pgd on (pgd.objoid=st.relid) WHERE c.ordinal_position=pgd.objsubid AND c.table_name=st.relname) AS "Comment" FROM information_schema.columns c LEFT JOIN (SELECT tc.table_schema, tc.table_name, cu.column_name, tc.constraint_type FROM information_schema.TABLE_CONSTRAINTS tc JOIN information_schema.KEY_COLUMN_USAGE  cu ON tc.table_schema=cu.table_schema and tc.table_name=cu.table_name and tc.constraint_name=cu.constraint_name and tc.constraint_type='PRIMARY KEY') pk ON pk.table_schema=c.table_schema AND pk.table_name=c.table_name AND pk.column_name=c.column_name WHERE c.table_name = 'frameworks' AND c.table_schema = 'public'
Executing (default): SELECT DISTINCT tc.constraint_name as constraint_name, tc.constraint_schema as constraint_schema, tc.constraint_catalog as constraint_catalog, tc.table_name as table_name,tc.table_schema as table_schema,tc.table_catalog as table_catalog,kcu.column_name as column_name,ccu.table_schema  AS referenced_table_schema,ccu.table_catalog  AS referenced_table_catalog,ccu.table_name  AS referenced_table_name,ccu.column_name AS referenced_column_name FROM information_schema.table_constraints AS tc JOIN information_schema.key_column_usage AS kcu ON tc.constraint_name = kcu.constraint_name JOIN information_schema.constraint_column_usage AS ccu ON ccu.constraint_name = tc.constraint_name WHERE constraint_type = 'FOREIGN KEY' AND tc.table_name = 'frameworks' AND tc.table_catalog = 'openpai'
Executing (default): SELECT pk.constraint_type as "Constraint",c.column_name as "Field", c.column_default as "Default",c.is_nullable as "Null", (CASE WHEN c.udt_name = 'hstore' THEN c.udt_name ELSE c.data_type END) || (CASE WHEN c.character_maximum_length IS NOT NULL THEN '(' || c.character_maximum_length || ')' ELSE '' END) as "Type", (SELECT array_agg(e.enumlabel) FROM pg_catalog.pg_type t JOIN pg_catalog.pg_enum e ON t.oid=e.enumtypid WHERE t.typname=c.udt_name) AS "special", (SELECT pgd.description FROM pg_catalog.pg_statio_all_tables AS st INNER JOIN pg_catalog.pg_description pgd on (pgd.objoid=st.relid) WHERE c.ordinal_position=pgd.objsubid AND c.table_name=st.relname) AS "Comment" FROM information_schema.columns c LEFT JOIN (SELECT tc.table_schema, tc.table_name, cu.column_name, tc.constraint_type FROM information_schema.TABLE_CONSTRAINTS tc JOIN information_schema.KEY_COLUMN_USAGE  cu ON tc.table_schema=cu.table_schema and tc.table_name=cu.table_name and tc.constraint_name=cu.constraint_name and tc.constraint_type='PRIMARY KEY') pk ON pk.table_schema=c.table_schema AND pk.table_name=c.table_name AND pk.column_name=c.column_name WHERE c.table_name = 'framework_history' AND c.table_schema = 'public'
Executing (default): SELECT DISTINCT tc.constraint_name as constraint_name, tc.constraint_schema as constraint_schema, tc.constraint_catalog as constraint_catalog, tc.table_name as table_name,tc.table_schema as table_schema,tc.table_catalog as table_catalog,kcu.column_name as column_name,ccu.table_schema  AS referenced_table_schema,ccu.table_catalog  AS referenced_table_catalog,ccu.table_name  AS referenced_table_name,ccu.column_name AS referenced_column_name FROM information_schema.table_constraints AS tc JOIN information_schema.key_column_usage AS kcu ON tc.constraint_name = kcu.constraint_name JOIN information_schema.constraint_column_usage AS ccu ON ccu.constraint_name = tc.constraint_name WHERE constraint_type = 'FOREIGN KEY' AND tc.table_name = 'framework_history' AND tc.table_catalog = 'openpai'
Executing (default): SELECT pk.constraint_type as "Constraint",c.column_name as "Field", c.column_default as "Default",c.is_nullable as "Null", (CASE WHEN c.udt_name = 'hstore' THEN c.udt_name ELSE c.data_type END) || (CASE WHEN c.character_maximum_length IS NOT NULL THEN '(' || c.character_maximum_length || ')' ELSE '' END) as "Type", (SELECT array_agg(e.enumlabel) FROM pg_catalog.pg_type t JOIN pg_catalog.pg_enum e ON t.oid=e.enumtypid WHERE t.typname=c.udt_name) AS "special", (SELECT pgd.description FROM pg_catalog.pg_statio_all_tables AS st INNER JOIN pg_catalog.pg_description pgd on (pgd.objoid=st.relid) WHERE c.ordinal_position=pgd.objsubid AND c.table_name=st.relname) AS "Comment" FROM information_schema.columns c LEFT JOIN (SELECT tc.table_schema, tc.table_name, cu.column_name, tc.constraint_type FROM information_schema.TABLE_CONSTRAINTS tc JOIN information_schema.KEY_COLUMN_USAGE  cu ON tc.table_schema=cu.table_schema and tc.table_name=cu.table_name and tc.constraint_name=cu.constraint_name and tc.constraint_type='PRIMARY KEY') pk ON pk.table_schema=c.table_schema AND pk.table_name=c.table_name AND pk.column_name=c.column_name WHERE c.table_name = 'pods' AND c.table_schema = 'public'
Executing (default): SELECT DISTINCT tc.constraint_name as constraint_name, tc.constraint_schema as constraint_schema, tc.constraint_catalog as constraint_catalog, tc.table_name as table_name,tc.table_schema as table_schema,tc.table_catalog as table_catalog,kcu.column_name as column_name,ccu.table_schema  AS referenced_table_schema,ccu.table_catalog  AS referenced_table_catalog,ccu.table_name  AS referenced_table_name,ccu.column_name AS referenced_column_name FROM information_schema.table_constraints AS tc JOIN information_schema.key_column_usage AS kcu ON tc.constraint_name = kcu.constraint_name JOIN information_schema.constraint_column_usage AS ccu ON ccu.constraint_name = tc.constraint_name WHERE constraint_type = 'FOREIGN KEY' AND tc.table_name = 'pods' AND tc.table_catalog = 'openpai'
Executing (default): SELECT pk.constraint_type as "Constraint",c.column_name as "Field", c.column_default as "Default",c.is_nullable as "Null", (CASE WHEN c.udt_name = 'hstore' THEN c.udt_name ELSE c.data_type END) || (CASE WHEN c.character_maximum_length IS NOT NULL THEN '(' || c.character_maximum_length || ')' ELSE '' END) as "Type", (SELECT array_agg(e.enumlabel) FROM pg_catalog.pg_type t JOIN pg_catalog.pg_enum e ON t.oid=e.enumtypid WHERE t.typname=c.udt_name) AS "special", (SELECT pgd.description FROM pg_catalog.pg_statio_all_tables AS st INNER JOIN pg_catalog.pg_description pgd on (pgd.objoid=st.relid) WHERE c.ordinal_position=pgd.objsubid AND c.table_name=st.relname) AS "Comment" FROM information_schema.columns c LEFT JOIN (SELECT tc.table_schema, tc.table_name, cu.column_name, tc.constraint_type FROM information_schema.TABLE_CONSTRAINTS tc JOIN information_schema.KEY_COLUMN_USAGE  cu ON tc.table_schema=cu.table_schema and tc.table_name=cu.table_name and tc.constraint_name=cu.constraint_name and tc.constraint_type='PRIMARY KEY') pk ON pk.table_schema=c.table_schema AND pk.table_name=c.table_name AND pk.column_name=c.column_name WHERE c.table_name = 'framework_events' AND c.table_schema = 'public'
Executing (default): SELECT DISTINCT tc.constraint_name as constraint_name, tc.constraint_schema as constraint_schema, tc.constraint_catalog as constraint_catalog, tc.table_name as table_name,tc.table_schema as table_schema,tc.table_catalog as table_catalog,kcu.column_name as column_name,ccu.table_schema  AS referenced_table_schema,ccu.table_catalog  AS referenced_table_catalog,ccu.table_name  AS referenced_table_name,ccu.column_name AS referenced_column_name FROM information_schema.table_constraints AS tc JOIN information_schema.key_column_usage AS kcu ON tc.constraint_name = kcu.constraint_name JOIN information_schema.constraint_column_usage AS ccu ON ccu.constraint_name = tc.constraint_name WHERE constraint_type = 'FOREIGN KEY' AND tc.table_name = 'framework_events' AND tc.table_catalog = 'openpai'
Executing (default): SELECT pk.constraint_type as "Constraint",c.column_name as "Field", c.column_default as "Default",c.is_nullable as "Null", (CASE WHEN c.udt_name = 'hstore' THEN c.udt_name ELSE c.data_type END) || (CASE WHEN c.character_maximum_length IS NOT NULL THEN '(' || c.character_maximum_length || ')' ELSE '' END) as "Type", (SELECT array_agg(e.enumlabel) FROM pg_catalog.pg_type t JOIN pg_catalog.pg_enum e ON t.oid=e.enumtypid WHERE t.typname=c.udt_name) AS "special", (SELECT pgd.description FROM pg_catalog.pg_statio_all_tables AS st INNER JOIN pg_catalog.pg_description pgd on (pgd.objoid=st.relid) WHERE c.ordinal_position=pgd.objsubid AND c.table_name=st.relname) AS "Comment" FROM information_schema.columns c LEFT JOIN (SELECT tc.table_schema, tc.table_name, cu.column_name, tc.constraint_type FROM information_schema.TABLE_CONSTRAINTS tc JOIN information_schema.KEY_COLUMN_USAGE  cu ON tc.table_schema=cu.table_schema and tc.table_name=cu.table_name and tc.constraint_name=cu.constraint_name and tc.constraint_type='PRIMARY KEY') pk ON pk.table_schema=c.table_schema AND pk.table_name=c.table_name AND pk.column_name=c.column_name WHERE c.table_name = 'pod_events' AND c.table_schema = 'public'
Executing (default): SELECT DISTINCT tc.constraint_name as constraint_name, tc.constraint_schema as constraint_schema, tc.constraint_catalog as constraint_catalog, tc.table_name as table_name,tc.table_schema as table_schema,tc.table_catalog as table_catalog,kcu.column_name as column_name,ccu.table_schema  AS referenced_table_schema,ccu.table_catalog  AS referenced_table_catalog,ccu.table_name  AS referenced_table_name,ccu.column_name AS referenced_column_name FROM information_schema.table_constraints AS tc JOIN information_schema.key_column_usage AS kcu ON tc.constraint_name = kcu.constraint_name JOIN information_schema.constraint_column_usage AS ccu ON ccu.constraint_name = tc.constraint_name WHERE constraint_type = 'FOREIGN KEY' AND tc.table_name = 'pod_events' AND tc.table_catalog = 'openpai'
Executing (default): SELECT pk.constraint_type as "Constraint",c.column_name as "Field", c.column_default as "Default",c.is_nullable as "Null", (CASE WHEN c.udt_name = 'hstore' THEN c.udt_name ELSE c.data_type END) || (CASE WHEN c.character_maximum_length IS NOT NULL THEN '(' || c.character_maximum_length || ')' ELSE '' END) as "Type", (SELECT array_agg(e.enumlabel) FROM pg_catalog.pg_type t JOIN pg_catalog.pg_enum e ON t.oid=e.enumtypid WHERE t.typname=c.udt_name) AS "special", (SELECT pgd.description FROM pg_catalog.pg_statio_all_tables AS st INNER JOIN pg_catalog.pg_description pgd on (pgd.objoid=st.relid) WHERE c.ordinal_position=pgd.objsubid AND c.table_name=st.relname) AS "Comment" FROM information_schema.columns c LEFT JOIN (SELECT tc.table_schema, tc.table_name, cu.column_name, tc.constraint_type FROM information_schema.TABLE_CONSTRAINTS tc JOIN information_schema.KEY_COLUMN_USAGE  cu ON tc.table_schema=cu.table_schema and tc.table_name=cu.table_name and tc.constraint_name=cu.constraint_name and tc.constraint_type='PRIMARY KEY') pk ON pk.table_schema=c.table_schema AND pk.table_name=c.table_name AND pk.column_name=c.column_name WHERE c.table_name = 'version' AND c.table_schema = 'public'
Executing (default): SELECT DISTINCT tc.constraint_name as constraint_name, tc.constraint_schema as constraint_schema, tc.constraint_catalog as constraint_catalog, tc.table_name as table_name,tc.table_schema as table_schema,tc.table_catalog as table_catalog,kcu.column_name as column_name,ccu.table_schema  AS referenced_table_schema,ccu.table_catalog  AS referenced_table_catalog,ccu.table_name  AS referenced_table_name,ccu.column_name AS referenced_column_name FROM information_schema.table_constraints AS tc JOIN information_schema.key_column_usage AS kcu ON tc.constraint_name = kcu.constraint_name JOIN information_schema.constraint_column_usage AS ccu ON ccu.constraint_name = tc.constraint_name WHERE constraint_type = 'FOREIGN KEY' AND tc.table_name = 'version' AND tc.table_catalog = 'openpai'
Executing (default): ALTER TABLE "framework_history" ALTER COLUMN "insertedAt" DROP NOT NULL;ALTER TABLE "framework_history" ALTER COLUMN "insertedAt" DROP DEFAULT;ALTER TABLE "framework_history" ALTER COLUMN "insertedAt" TYPE TIMESTAMP WITH TIME ZONE;
Executing (default): ALTER TABLE "pod_events" ALTER COLUMN "insertedAt" DROP NOT NULL;ALTER TABLE "pod_events" ALTER COLUMN "insertedAt" DROP DEFAULT;ALTER TABLE "pod_events" ALTER COLUMN "insertedAt" TYPE TIMESTAMP WITH TIME ZONE;
Executing (default): ALTER TABLE "framework_events" ALTER COLUMN "insertedAt" DROP NOT NULL;ALTER TABLE "framework_events" ALTER COLUMN "insertedAt" DROP DEFAULT;ALTER TABLE "framework_events" ALTER COLUMN "insertedAt" TYPE TIMESTAMP WITH TIME ZONE;
Executing (default): ALTER TABLE "frameworks" ALTER COLUMN "insertedAt" DROP NOT NULL;ALTER TABLE "frameworks" ALTER COLUMN "insertedAt" DROP DEFAULT;ALTER TABLE "frameworks" ALTER COLUMN "insertedAt" TYPE TIMESTAMP WITH TIME ZONE;
Executing (default): ALTER TABLE "framework_history" DROP CONSTRAINT "framework_history_frameworkName_fkey"
Executing (default): ALTER TABLE "version" ALTER COLUMN "version" DROP NOT NULL;ALTER TABLE "version" ALTER COLUMN "version" DROP DEFAULT;ALTER TABLE "version" ALTER COLUMN "version" TYPE VARCHAR(36);
Executing (default): ALTER TABLE "pods" ALTER COLUMN "insertedAt" DROP NOT NULL;ALTER TABLE "pods" ALTER COLUMN "insertedAt" DROP DEFAULT;ALTER TABLE "pods" ALTER COLUMN "insertedAt" TYPE TIMESTAMP WITH TIME ZONE;
Executing (default): ALTER TABLE "pod_events" DROP CONSTRAINT "pod_events_frameworkName_fkey"
Executing (default): ALTER TABLE "framework_events" DROP CONSTRAINT "framework_events_frameworkName_fkey"
Executing (default): ALTER TABLE "frameworks" ALTER COLUMN "namespace" DROP NOT NULL;ALTER TABLE "frameworks" ALTER COLUMN "namespace" DROP DEFAULT;ALTER TABLE "frameworks" ALTER COLUMN "namespace" TYPE VARCHAR(64);
Executing (default): ALTER TABLE "version" ALTER COLUMN "commitVersion" DROP NOT NULL;ALTER TABLE "version" ALTER COLUMN "commitVersion" DROP DEFAULT;ALTER TABLE "version" ALTER COLUMN "commitVersion" TYPE VARCHAR(64);
Executing (default): ALTER TABLE "pods" DROP CONSTRAINT "pods_frameworkName_fkey"
Executing (default): ALTER TABLE "framework_history" ALTER COLUMN "frameworkName" SET NOT NULL;ALTER TABLE "framework_history"  ADD FOREIGN KEY ("frameworkName") REFERENCES "frameworks" ("name") ON DELETE CASCADE ON UPDATE CASCADE;
Executing (default): ALTER TABLE "version" ALTER COLUMN "createdAt" SET NOT NULL;ALTER TABLE "version" ALTER COLUMN "createdAt" DROP DEFAULT;ALTER TABLE "version" ALTER COLUMN "createdAt" TYPE TIMESTAMP WITH TIME ZONE;
Executing (default): ALTER TABLE "pod_events" ALTER COLUMN "frameworkName" SET NOT NULL;ALTER TABLE "pod_events"  ADD FOREIGN KEY ("frameworkName") REFERENCES "frameworks" ("name") ON DELETE CASCADE ON UPDATE CASCADE;
Executing (default): ALTER TABLE "version" ALTER COLUMN "updatedAt" SET NOT NULL;ALTER TABLE "version" ALTER COLUMN "updatedAt" DROP DEFAULT;ALTER TABLE "version" ALTER COLUMN "updatedAt" TYPE TIMESTAMP WITH TIME ZONE;
Executing (default): ALTER TABLE "framework_events" ALTER COLUMN "frameworkName" SET NOT NULL;ALTER TABLE "framework_events"  ADD FOREIGN KEY ("frameworkName") REFERENCES "frameworks" ("name") ON DELETE CASCADE ON UPDATE CASCADE;
Executing (default): SELECT i.relname AS name, ix.indisprimary AS primary, ix.indisunique AS unique, ix.indkey AS indkey, array_agg(a.attnum) as column_indexes, array_agg(a.attname) AS column_names, pg_get_indexdef(ix.indexrelid) AS definition FROM pg_class t, pg_class i, pg_index ix, pg_attribute a WHERE t.oid = ix.indrelid AND i.oid = ix.indexrelid AND a.attrelid = t.oid AND t.relkind = 'r' and t.relname = 'version' GROUP BY i.relname, ix.indexrelid, ix.indisprimary, ix.indisunique, ix.indkey ORDER BY i.relname;
Executing (default): ALTER TABLE "frameworks" ALTER COLUMN "jobName" DROP NOT NULL;ALTER TABLE "frameworks" ALTER COLUMN "jobName" DROP DEFAULT;ALTER TABLE "frameworks" ALTER COLUMN "jobName" TYPE VARCHAR(256);
Executing (default): ALTER TABLE "pods" ALTER COLUMN "frameworkName" SET NOT NULL;ALTER TABLE "pods"  ADD FOREIGN KEY ("frameworkName") REFERENCES "frameworks" ("name") ON DELETE CASCADE ON UPDATE CASCADE;
Executing (default): ALTER TABLE "framework_history" ALTER COLUMN "attemptIndex" DROP NOT NULL;ALTER TABLE "framework_history" ALTER COLUMN "attemptIndex" DROP DEFAULT;ALTER TABLE "framework_history" ALTER COLUMN "attemptIndex" TYPE INTEGER;
Executing (default): ALTER TABLE "framework_history" ALTER COLUMN "historyType" SET NOT NULL;ALTER TABLE "framework_history" ALTER COLUMN "historyType" SET DEFAULT 'retry';ALTER TABLE "framework_history" ALTER COLUMN "historyType" TYPE VARCHAR(16);
Executing (default): ALTER TABLE "pod_events" ALTER COLUMN "podUid" SET NOT NULL;ALTER TABLE "pod_events" ALTER COLUMN "podUid" DROP DEFAULT;ALTER TABLE "pod_events" ALTER COLUMN "podUid" TYPE VARCHAR(36);
Executing (default): ALTER TABLE "framework_history" ALTER COLUMN "snapshot" DROP NOT NULL;ALTER TABLE "framework_history" ALTER COLUMN "snapshot" DROP DEFAULT;ALTER TABLE "framework_history" ALTER COLUMN "snapshot" TYPE TEXT;
Executing (default): ALTER TABLE "framework_events" ALTER COLUMN "type" SET NOT NULL;ALTER TABLE "framework_events" ALTER COLUMN "type" DROP DEFAULT;ALTER TABLE "framework_events" ALTER COLUMN "type" TYPE VARCHAR(32);
Executing (default): ALTER TABLE "pod_events" ALTER COLUMN "type" SET NOT NULL;ALTER TABLE "pod_events" ALTER COLUMN "type" DROP DEFAULT;ALTER TABLE "pod_events" ALTER COLUMN "type" TYPE VARCHAR(32);
Executing (default): ALTER TABLE "framework_history" ALTER COLUMN "updatedAt" SET NOT NULL;ALTER TABLE "framework_history" ALTER COLUMN "updatedAt" DROP DEFAULT;ALTER TABLE "framework_history" ALTER COLUMN "updatedAt" TYPE TIMESTAMP WITH TIME ZONE;
Executing (default): ALTER TABLE "pod_events" ALTER COLUMN "message" DROP NOT NULL;ALTER TABLE "pod_events" ALTER COLUMN "message" DROP DEFAULT;ALTER TABLE "pod_events" ALTER COLUMN "message" TYPE TEXT;
Executing (default): ALTER TABLE "frameworks" ALTER COLUMN "userName" DROP NOT NULL;ALTER TABLE "frameworks" ALTER COLUMN "userName" DROP DEFAULT;ALTER TABLE "frameworks" ALTER COLUMN "userName" TYPE VARCHAR(256);
Executing (default): ALTER TABLE "framework_events" ALTER COLUMN "message" DROP NOT NULL;ALTER TABLE "framework_events" ALTER COLUMN "message" DROP DEFAULT;ALTER TABLE "framework_events" ALTER COLUMN "message" TYPE TEXT;
Executing (default): SELECT i.relname AS name, ix.indisprimary AS primary, ix.indisunique AS unique, ix.indkey AS indkey, array_agg(a.attnum) as column_indexes, array_agg(a.attname) AS column_names, pg_get_indexdef(ix.indexrelid) AS definition FROM pg_class t, pg_class i, pg_index ix, pg_attribute a WHERE t.oid = ix.indrelid AND i.oid = ix.indexrelid AND a.attrelid = t.oid AND t.relkind = 'r' and t.relname = 'framework_history' GROUP BY i.relname, ix.indexrelid, ix.indisprimary, ix.indisunique, ix.indkey ORDER BY i.relname;
Executing (default): ALTER TABLE "framework_events" ALTER COLUMN "event" DROP NOT NULL;ALTER TABLE "framework_events" ALTER COLUMN "event" DROP DEFAULT;ALTER TABLE "framework_events" ALTER COLUMN "event" TYPE TEXT;
Executing (default): ALTER TABLE "pods" ALTER COLUMN "attemptIndex" DROP NOT NULL;ALTER TABLE "pods" ALTER COLUMN "attemptIndex" DROP DEFAULT;ALTER TABLE "pods" ALTER COLUMN "attemptIndex" TYPE INTEGER;
Executing (default): ALTER TABLE "pod_events" ALTER COLUMN "event" DROP NOT NULL;ALTER TABLE "pod_events" ALTER COLUMN "event" DROP DEFAULT;ALTER TABLE "pod_events" ALTER COLUMN "event" TYPE TEXT;
Executing (default): ALTER TABLE "frameworks" ALTER COLUMN "jobConfig" DROP NOT NULL;ALTER TABLE "frameworks" ALTER COLUMN "jobConfig" DROP DEFAULT;ALTER TABLE "frameworks" ALTER COLUMN "jobConfig" TYPE TEXT;
Executing (default): ALTER TABLE "pod_events" ALTER COLUMN "updatedAt" SET NOT NULL;ALTER TABLE "pod_events" ALTER COLUMN "updatedAt" DROP DEFAULT;ALTER TABLE "pod_events" ALTER COLUMN "updatedAt" TYPE TIMESTAMP WITH TIME ZONE;
Executing (default): ALTER TABLE "framework_events" ALTER COLUMN "updatedAt" SET NOT NULL;ALTER TABLE "framework_events" ALTER COLUMN "updatedAt" DROP DEFAULT;ALTER TABLE "framework_events" ALTER COLUMN "updatedAt" TYPE TIMESTAMP WITH TIME ZONE;
Executing (default): ALTER TABLE "pods" ALTER COLUMN "taskroleName" DROP NOT NULL;ALTER TABLE "pods" ALTER COLUMN "taskroleName" DROP DEFAULT;ALTER TABLE "pods" ALTER COLUMN "taskroleName" TYPE VARCHAR(256);
Executing (default): ALTER TABLE "frameworks" ALTER COLUMN "executionType" DROP NOT NULL;ALTER TABLE "frameworks" ALTER COLUMN "executionType" DROP DEFAULT;ALTER TABLE "frameworks" ALTER COLUMN "executionType" TYPE VARCHAR(32);
Executing (default): SELECT i.relname AS name, ix.indisprimary AS primary, ix.indisunique AS unique, ix.indkey AS indkey, array_agg(a.attnum) as column_indexes, array_agg(a.attname) AS column_names, pg_get_indexdef(ix.indexrelid) AS definition FROM pg_class t, pg_class i, pg_index ix, pg_attribute a WHERE t.oid = ix.indrelid AND i.oid = ix.indexrelid AND a.attrelid = t.oid AND t.relkind = 'r' and t.relname = 'framework_events' GROUP BY i.relname, ix.indexrelid, ix.indisprimary, ix.indisunique, ix.indkey ORDER BY i.relname;
Executing (default): ALTER TABLE "pods" ALTER COLUMN "taskroleIndex" DROP NOT NULL;ALTER TABLE "pods" ALTER COLUMN "taskroleIndex" DROP DEFAULT;ALTER TABLE "pods" ALTER COLUMN "taskroleIndex" TYPE INTEGER;
Executing (default): SELECT i.relname AS name, ix.indisprimary AS primary, ix.indisunique AS unique, ix.indkey AS indkey, array_agg(a.attnum) as column_indexes, array_agg(a.attname) AS column_names, pg_get_indexdef(ix.indexrelid) AS definition FROM pg_class t, pg_class i, pg_index ix, pg_attribute a WHERE t.oid = ix.indrelid AND i.oid = ix.indexrelid AND a.attrelid = t.oid AND t.relkind = 'r' and t.relname = 'pod_events' GROUP BY i.relname, ix.indexrelid, ix.indisprimary, ix.indisunique, ix.indkey ORDER BY i.relname;
Executing (default): ALTER TABLE "frameworks" ALTER COLUMN "creationTime" DROP NOT NULL;ALTER TABLE "frameworks" ALTER COLUMN "creationTime" DROP DEFAULT;ALTER TABLE "frameworks" ALTER COLUMN "creationTime" TYPE TIMESTAMP WITH TIME ZONE;
Executing (default): ALTER TABLE "pods" ALTER COLUMN "taskAttemptIndex" DROP NOT NULL;ALTER TABLE "pods" ALTER COLUMN "taskAttemptIndex" DROP DEFAULT;ALTER TABLE "pods" ALTER COLUMN "taskAttemptIndex" TYPE INTEGER;
Executing (default): ALTER TABLE "frameworks" ALTER COLUMN "virtualCluster" DROP NOT NULL;ALTER TABLE "frameworks" ALTER COLUMN "virtualCluster" DROP DEFAULT;ALTER TABLE "frameworks" ALTER COLUMN "virtualCluster" TYPE VARCHAR(256);
Executing (default): ALTER TABLE "pods" ALTER COLUMN "snapshot" DROP NOT NULL;ALTER TABLE "pods" ALTER COLUMN "snapshot" DROP DEFAULT;ALTER TABLE "pods" ALTER COLUMN "snapshot" TYPE TEXT;
Executing (default): ALTER TABLE "frameworks" ALTER COLUMN "jobPriority" DROP NOT NULL;ALTER TABLE "frameworks" ALTER COLUMN "jobPriority" DROP DEFAULT;ALTER TABLE "frameworks" ALTER COLUMN "jobPriority" TYPE VARCHAR(256);
Executing (default): ALTER TABLE "pods" ALTER COLUMN "updatedAt" SET NOT NULL;ALTER TABLE "pods" ALTER COLUMN "updatedAt" DROP DEFAULT;ALTER TABLE "pods" ALTER COLUMN "updatedAt" TYPE TIMESTAMP WITH TIME ZONE;
Executing (default): ALTER TABLE "frameworks" ALTER COLUMN "totalGpuNumber" DROP NOT NULL;ALTER TABLE "frameworks" ALTER COLUMN "totalGpuNumber" DROP DEFAULT;ALTER TABLE "frameworks" ALTER COLUMN "totalGpuNumber" TYPE INTEGER;
Executing (default): SELECT i.relname AS name, ix.indisprimary AS primary, ix.indisunique AS unique, ix.indkey AS indkey, array_agg(a.attnum) as column_indexes, array_agg(a.attname) AS column_names, pg_get_indexdef(ix.indexrelid) AS definition FROM pg_class t, pg_class i, pg_index ix, pg_attribute a WHERE t.oid = ix.indrelid AND i.oid = ix.indexrelid AND a.attrelid = t.oid AND t.relkind = 'r' and t.relname = 'pods' GROUP BY i.relname, ix.indexrelid, ix.indisprimary, ix.indisunique, ix.indkey ORDER BY i.relname;
Executing (default): ALTER TABLE "frameworks" ALTER COLUMN "totalTaskNumber" DROP NOT NULL;ALTER TABLE "frameworks" ALTER COLUMN "totalTaskNumber" DROP DEFAULT;ALTER TABLE "frameworks" ALTER COLUMN "totalTaskNumber" TYPE INTEGER;
Executing (default): ALTER TABLE "frameworks" ALTER COLUMN "totalTaskRoleNumber" DROP NOT NULL;ALTER TABLE "frameworks" ALTER COLUMN "totalTaskRoleNumber" DROP DEFAULT;ALTER TABLE "frameworks" ALTER COLUMN "totalTaskRoleNumber" TYPE INTEGER;
Executing (default): ALTER TABLE "frameworks" ALTER COLUMN "logPathInfix" DROP NOT NULL;ALTER TABLE "frameworks" ALTER COLUMN "logPathInfix" DROP DEFAULT;ALTER TABLE "frameworks" ALTER COLUMN "logPathInfix" TYPE VARCHAR(256);
Executing (default): ALTER TABLE "frameworks" ALTER COLUMN "submissionTime" SET NOT NULL;ALTER TABLE "frameworks" ALTER COLUMN "submissionTime" DROP DEFAULT;ALTER TABLE "frameworks" ALTER COLUMN "submissionTime" TYPE TIMESTAMP WITH TIME ZONE;
Executing (default): ALTER TABLE "frameworks" ALTER COLUMN "dockerSecretDef" DROP NOT NULL;ALTER TABLE "frameworks" ALTER COLUMN "dockerSecretDef" DROP DEFAULT;ALTER TABLE "frameworks" ALTER COLUMN "dockerSecretDef" TYPE TEXT;
Executing (default): ALTER TABLE "frameworks" ALTER COLUMN "configSecretDef" DROP NOT NULL;ALTER TABLE "frameworks" ALTER COLUMN "configSecretDef" DROP DEFAULT;ALTER TABLE "frameworks" ALTER COLUMN "configSecretDef" TYPE TEXT;
Executing (default): ALTER TABLE "frameworks" ALTER COLUMN "priorityClassDef" DROP NOT NULL;ALTER TABLE "frameworks" ALTER COLUMN "priorityClassDef" DROP DEFAULT;ALTER TABLE "frameworks" ALTER COLUMN "priorityClassDef" TYPE TEXT;
Executing (default): ALTER TABLE "frameworks" ALTER COLUMN "retries" DROP NOT NULL;ALTER TABLE "frameworks" ALTER COLUMN "retries" DROP DEFAULT;ALTER TABLE "frameworks" ALTER COLUMN "retries" TYPE INTEGER;
Executing (default): ALTER TABLE "frameworks" ALTER COLUMN "retryDelayTime" DROP NOT NULL;ALTER TABLE "frameworks" ALTER COLUMN "retryDelayTime" DROP DEFAULT;ALTER TABLE "frameworks" ALTER COLUMN "retryDelayTime" TYPE INTEGER;
Executing (default): ALTER TABLE "frameworks" ALTER COLUMN "platformRetries" DROP NOT NULL;ALTER TABLE "frameworks" ALTER COLUMN "platformRetries" DROP DEFAULT;ALTER TABLE "frameworks" ALTER COLUMN "platformRetries" TYPE INTEGER;
Executing (default): ALTER TABLE "frameworks" ALTER COLUMN "resourceRetries" DROP NOT NULL;ALTER TABLE "frameworks" ALTER COLUMN "resourceRetries" DROP DEFAULT;ALTER TABLE "frameworks" ALTER COLUMN "resourceRetries" TYPE INTEGER;
Executing (default): ALTER TABLE "frameworks" ALTER COLUMN "userRetries" DROP NOT NULL;ALTER TABLE "frameworks" ALTER COLUMN "userRetries" DROP DEFAULT;ALTER TABLE "frameworks" ALTER COLUMN "userRetries" TYPE INTEGER;
Executing (default): ALTER TABLE "frameworks" ALTER COLUMN "completionTime" DROP NOT NULL;ALTER TABLE "frameworks" ALTER COLUMN "completionTime" DROP DEFAULT;ALTER TABLE "frameworks" ALTER COLUMN "completionTime" TYPE TIMESTAMP WITH TIME ZONE;
Executing (default): ALTER TABLE "frameworks" ALTER COLUMN "appExitCode" DROP NOT NULL;ALTER TABLE "frameworks" ALTER COLUMN "appExitCode" DROP DEFAULT;ALTER TABLE "frameworks" ALTER COLUMN "appExitCode" TYPE INTEGER;
Executing (default): ALTER TABLE "frameworks" ALTER COLUMN "subState" DROP NOT NULL;ALTER TABLE "frameworks" ALTER COLUMN "subState" DROP DEFAULT;ALTER TABLE "frameworks" ALTER COLUMN "subState" TYPE VARCHAR(32);
Executing (default): ALTER TABLE "frameworks" ALTER COLUMN "state" DROP NOT NULL;ALTER TABLE "frameworks" ALTER COLUMN "state" DROP DEFAULT;ALTER TABLE "frameworks" ALTER COLUMN "state" TYPE VARCHAR(32);
Executing (default): ALTER TABLE "frameworks" ALTER COLUMN "snapshot" DROP NOT NULL;ALTER TABLE "frameworks" ALTER COLUMN "snapshot" DROP DEFAULT;ALTER TABLE "frameworks" ALTER COLUMN "snapshot" TYPE TEXT;
Executing (default): ALTER TABLE "frameworks" ALTER COLUMN "requestSynced" SET NOT NULL;ALTER TABLE "frameworks" ALTER COLUMN "requestSynced" SET DEFAULT false;ALTER TABLE "frameworks" ALTER COLUMN "requestSynced" TYPE BOOLEAN;
Executing (default): ALTER TABLE "frameworks" ALTER COLUMN "apiServerDeleted" SET NOT NULL;ALTER TABLE "frameworks" ALTER COLUMN "apiServerDeleted" SET DEFAULT false;ALTER TABLE "frameworks" ALTER COLUMN "apiServerDeleted" TYPE BOOLEAN;
Executing (default): ALTER TABLE "frameworks" ALTER COLUMN "archived" SET NOT NULL;ALTER TABLE "frameworks" ALTER COLUMN "archived" SET DEFAULT false;ALTER TABLE "frameworks" ALTER COLUMN "archived" TYPE BOOLEAN;
Executing (default): ALTER TABLE "frameworks" ALTER COLUMN "updatedAt" SET NOT NULL;ALTER TABLE "frameworks" ALTER COLUMN "updatedAt" DROP DEFAULT;ALTER TABLE "frameworks" ALTER COLUMN "updatedAt" TYPE TIMESTAMP WITH TIME ZONE;
Executing (default): SELECT i.relname AS name, ix.indisprimary AS primary, ix.indisunique AS unique, ix.indkey AS indkey, array_agg(a.attnum) as column_indexes, array_agg(a.attname) AS column_names, pg_get_indexdef(ix.indexrelid) AS definition FROM pg_class t, pg_class i, pg_index ix, pg_attribute a WHERE t.oid = ix.indrelid AND i.oid = ix.indexrelid AND a.attrelid = t.oid AND t.relkind = 'r' and t.relname = 'frameworks' GROUP BY i.relname, ix.indexrelid, ix.indisprimary, ix.indisunique, ix.indkey ORDER BY i.relname;
2020-10-15T01:14:44.215Z [ERROR]  
meta = {
  "message": "HTTP request failed",
  "stack": "HttpError: HTTP request failed\n    at Request.localVarRequest [as _callback] (/database-controller/src/node_modules/@kubernetes/client-node/dist/gen/api/customObjectsApi.js:1244:40)\n    at Request.self.callback (/database-controller/src/node_modules/request/request.js:185:22)\n    at emitTwo (events.js:126:13)\n    at Request.emit (events.js:214:7)\n    at Request.<anonymous> (/database-controller/src/node_modules/request/request.js:1154:10)\n    at emitOne (events.js:116:13)\n    at Request.emit (events.js:211:7)\n    at IncomingMessage.<anonymous> (/database-controller/src/node_modules/request/request.js:1076:12)\n    at Object.onceWrapper (events.js:313:30)\n    at emitNone (events.js:111:20)\n    at IncomingMessage.emit (events.js:208:7)\n    at endReadableNT (_stream_readable.js:1064:12)\n    at _combinedTickCallback (internal/process/next_tick.js:139:11)\n    at process._tickCallback (internal/process/next_tick.js:181:9)",
  "response": {
    "statusCode": 404,
    "body": "404 page not found\n",
    "headers": {
      "content-type": "text/plain; charset=utf-8",
      "x-content-type-options": "nosniff",
      "date": "Thu, 15 Oct 2020 01:14:44 GMT",
      "content-length": "19",
      "connection": "close"
    },
    "request": {
      "uri": {
        "protocol": "https:",
        "slashes": true,
        "auth": null,
        "host": "10.192.0.1:443",
        "port": "443",
        "hostname": "10.192.0.1",
        "hash": null,
        "search": null,
        "query": null,
        "pathname": "/apis/frameworkcontroller.microsoft.com/v1/namespaces/default/frameworks",
        "path": "/apis/frameworkcontroller.microsoft.com/v1/namespaces/default/frameworks",
        "href": "https://10.192.0.1:443/apis/frameworkcontroller.microsoft.com/v1/namespaces/default/frameworks"
      },
      "method": "GET",
      "headers": {
        "Accept": "application/json",
        "Authorization": "Bearer eyJhbGciOiJSUzI1NiIsImtpZCI6IiJ9.eyJpc3MiOiJrdWJlcm5ldGVzL3NlcnZpY2VhY2NvdW50Iiwia3ViZXJuZXRlcy5pby9zZXJ2aWNlYWNjb3VudC9uYW1lc3BhY2UiOiJkZWZhdWx0Iiwia3ViZXJuZXRlcy5pby9zZXJ2aWNlYWNjb3VudC9zZWNyZXQubmFtZSI6ImRhdGFiYXNlLWNvbnRyb2xsZXItYWNjb3VudC10b2tlbi1sbm16OCIsImt1YmVybmV0ZXMuaW8vc2VydmljZWFjY291bnQvc2VydmljZS1hY2NvdW50Lm5hbWUiOiJkYXRhYmFzZS1jb250cm9sbGVyLWFjY291bnQiLCJrdWJlcm5ldGVzLmlvL3NlcnZpY2VhY2NvdW50L3NlcnZpY2UtYWNjb3VudC51aWQiOiI2NTM4MmIzMy1iM2FmLTQ3ZWEtYThiYi0zNzk0MDAxMzk5ZTgiLCJzdWIiOiJzeXN0ZW06c2VydmljZWFjY291bnQ6ZGVmYXVsdDpkYXRhYmFzZS1jb250cm9sbGVyLWFjY291bnQifQ.UCdgW4LZFyX0GxNFUpeXLCcgD8nZZhKI_2tAjcbhV4lDOXN0CpyiMT5933c5vMA2od7yEhklUtQir25tEyCJQropaJsrnycDyez8EbrkUaAVEzVZq0r7Tem08ZMI23v1cbLCQKwXC2GKK1lbuPr36h7xmHnh4Iv4c5bujSlshLAQIywHp-pb60BimTYVhdS2V0DMEofkxmXGI3efBvzcQfibtvmpO87aQfWC3ThQwhZRFWBbL52CTooQjN2ifYlKjuHrg1dfgilO_rwkUbXbC5cm5thdMxOkH_GmgJK2ibzG8mF2SWyGasXzak_4KSgMrdmH5DMp64iqQnGU_ZGVmg"
      }
    }
  },
  "body": "404 page not found\n",
  "statusCode": 404,
  "name": "HttpError"
}
```

From this point, I'm not sure what to check further. Any help with this?

--- a/src/database-controller/deploy/service.yaml
+++ b/src/database-controller/deploy/service.yaml
@@ -5,6 +5,7 @@ cluster-type:
   - k8s
 
 prerequisite:
+  - frameworkcontroller
   - postgresql
 
 template-list:

